force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +380,13 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        Integer agePerson1 = person1.getAge();
+        Integer agePerson2 = person2.getAge();
+        if(agePerson1 > agePerson2) {
+            return person1;
+        } else if(agePerson1 == agePerson2) {
+            return person1;
+        } else
+        return person2;
```

Comment:
Think about how you could shorten up your code by simply using `>=` in the first if statement.